pyproject.toml

```
@@ -1,8 +1,34 @@
 [tool.poetry]
 name = "sqlalchemy-ibmi"
-version = "0.1.0"
-description = "'SQLAlchemy support for IBM i db2'"
-authors = ["Naveen-Ram <naveen.ram@ibm.com>"]
+version = "0.9.0"
+description = "SQLAlchemy support for Db2 on IBM i"
+readme = "README.md"
+author = [
```

```suggestion
authors = [
```

https://python-poetry.org/docs/pyproject/#authors